src/mo_frontend/typing.mli

```diff
@@ -13,3 +13,4 @@ val check_actors : ?viper_mode:bool -> ?check_actors:bool -> scope -> Syntax.pro
 val check_stab_sig : scope -> Syntax.stab_sig -> (field list) Diag.result
 
 val heartbeat_type : typ
+val global_timer_set_type : typ
```

Comment:
Maybe these should live in Type.ml? I think there are other well-known types in there.

Reply:
I tend to agree...

Reply:
Done: 980cd23c8